django/thunderstore/core/api_urls.py

```diff
@@ -34,7 +34,7 @@
 ]
 
 api_urls = [
-    path("v1/", include((v1_urls, "v1"), namespace="v1")),
+    path("v1/", include((v1_urls, "v1"))),
```

Comment:
Why was namespace removed?

Reply:
Removed it here so I could add to community/urls.py line 10. I also forgot to add it there 😅